src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.commands;
+
+
+/**
+ * Sorts all people in the address book by name in ascneding/descending lexicographical order.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
+            + "Sort all persons in the address book in lexicographical order.\n\t"
+            + "Parameters: sort [ascending | asc] | [descending | desc] \n\t"
+            + "Example: " + COMMAND_WORD + "desc";
+
+    public static final String MESSAGE_SUCCESS_ASCENDING = "Address book has been sorted in lexicographical order!";
+    public static final String MESSAGE_SUCCESS_DESCENDING = "Address book has been sorted in colexicographical order!";
+    private boolean ascending = true;
```

Comment:
Nitpick on Java standards. Variable should be isAscending, as ascending is not a pure yes/no.
Also, it is a bit of a bad practice to dictate default (function) behaviours arbitrarily (always true unless otherwise). It would be more acceptable at a user level, but at a code level, it should be as clear/simple as possible without having to remember the default behaviour.